### DISCUSSION TITLE
A way to see component styles the Tailwind way?

### DISCUSSION THREAD

**Discussion by ddahan** [saadeghi/daisyui#719] [Q&A] [answered]
Hi, 

I just found out Daisy UI and love the concept.
However, the main reason I decided to use low-level tools like Tailwind (rather than UI component libs like Bootstrap or Ionic) is to be able to see the component styles directly in my code.
Even if the html is more verbose that way, I know directly what's happening and can change the behaviour.

Is there a way to see the default component styles when working with daisy UI? Of course, I mean somewhere else than in the chrome inspector. It would be best of both worlds to me: not having to reinvent the wheel by creating my own buttons + having a full control over the applied styles.

For example, when writing `<button class="btn">Button</button>` i'd like to know that my button is rounded, the hover behaviour, etc. And since I have no real CSS knowledge and use Tailwind CSS classes only, I'd like to see the styles the tailwind way (for example: I'd like to see `rounded-sm` rather than `border-radius: 0.125rem`).

Is there any way to do this? Thanks.

**Comment by saadeghi** [comment, marked as answer]
If you use [Tailwind CSS IntelliSense](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss) hovering on the `btn` class would show the style of `btn`  
That would be the best way to see what's happening under the hood.

**Reply by ddahan** [reply]
Oh ok thanks for this, I didn't know. I'll try it as soon as possible.
